app/src/main/java/com/example/georgesamuel/dubaihotels/entities/HotelsDetailsModel.java

```diff
@@ -0,0 +1,24 @@
+
+package com.example.georgesamuel.dubaihotels.entities;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+import java.io.Serializable;
+import java.util.List;
+
+public class HotelsDetailsModel implements Serializable {
```

Comment:
class name is confusing , i though its details object .. rename to HotelsResponse 

you can use Pojo generator plugin.

https://github.com/robohorse/RoboPOJOGenerator

Reply:
ok , i checked it